specification/cognitiveservices/Language.Conversations/tsp-output/@azure-tools/typespec-autorest/azure.ai.language.conversations.json

```diff
@@ -46,7 +46,47 @@
     }
   },
   "tags": [],
-  "paths": {},
```

Comment:
This is incorrect output location. Please use the standard config in the other comment and check in the swagger files in the correct location under  ./data-plane/Lanauge.Conversation/xxx